WordPress/Classes/Utility/Universal Links/Routes+Jetpack.swift

```diff
@@ -12,3 +12,35 @@ extension JetpackRoute: NavigationAction {
         // If we handle deferred linking only then it would be relevant.
     }
 }
+
+// MARK: - Trigger Export on WordPress
+
+struct WordPressExportRoute: Route {
+    let path = "/export-213"
+    let section: DeepLinkSection? = nil
+    var action: NavigationAction {
+        return self
+    }
+}
+
+extension WordPressExportRoute: NavigationAction {
+    func perform(_ values: [String: String], source: UIViewController?, router: LinkRouter) {
+        // Trigger the export process
+        ContentMigrationCoordinator.shared.startAndDo { _ in
+            // Regardless of the result, redirect the user back to Jetpack.
+            let jetpackUrl: URL? = {
+                var components = URLComponents()
+                components.scheme = JetpackNotificationMigrationService.jetpackScheme
```

Comment:
I should probably change this to something that's less dependant on the notification scheme.

Reply:
On second thought, changing this to `jetpack://` might inadvertently open the game instead 😆 . So I'll just keep this as is for now.